Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -286,18 +291,6 @@ private void finishDropInWithPendingResult(boolean cancelPendingActivityAnimatio
         }
     }
 
-    private void updateVaultedPaymentMethodNonces(boolean refetch) {
```

Comment:
We removed this because it was only being called immediately before `onError` which dismisses drop-in. Since we are re-fetching the vaulted payment methods in `onCreate`, it doesn't make sense to update these here.